controllers/etcdcluster_controller_test.go

```diff
@@ -96,6 +96,18 @@ func (s *StaticResponseEtcdAPI) Remove(ctx context.Context, mID string) error {
 	return fmt.Errorf("unknown member: %s", mID)
 }
 
+func (s *StaticResponseEtcdAPI) Defragment(ctx context.Context, m etcd.Member) error {
+	s.Lock()
```

Comment:
Could you please describe why we need this locking thing?

Reply:
We don't actually need it for these mock functions, but other empty implementations for this mock have it, so I've continued it